tests/integration/linux_bridge_test.py

```diff
@@ -269,6 +278,23 @@ def test_replace_port_on_linux_bridge(port0_vlan101, port1_up):
         )
 
 
+def test_linux_bridge_over_bond_over_slave(bond0):
+    bridge_name = TEST_BRIDGE0
+    bond_name = bond0[Interface.KEY][0][Interface.NAME]
+    bridge_config_state = _create_bridge_subtree_config((bond_name,))
+    with linux_bridge(
+        bridge_name, bridge_config_state, create=False
+    ) as bridge0:
+        desired_state = deepcopy(bond0)
+        bridge_state = bridge0[Interface.KEY][0]
+        desired_state[Interface.KEY].append(bridge_state)
+        libnmstate.apply(desired_state)
+
+        assertlib.assert_state_match(desired_state)
+
+    assertlib.assert_absent(bridge_name)
```

Comment:
I guess it is as needed there as in https://github.com/nmstate/nmstate/pull/489/files#diff-41e570df111ba3cf748cf4f22fb0e3c1R240 . 

Just say the word, I'll remove it. But, I think it can't hurt to assert that the bridge is gone - but I understand the reasoning that it might lead the reader into thinking that's somehow important to the test.